lib/index.js

```diff
@@ -67,6 +68,12 @@ function preprocessor(css, options, filename) {
     return settings ? plugin(settings) : plugin;
   });
 
+  if (options.stylesIsolation) {
+    var isolation = require('./isolation');
+    var isolate = [isolation.resetGeneric, isolation.resetInherited];
+    plugins = isolate.concat(plugins);
```

Comment:
We might need to be careful where we put these plugins. I think they should run after `postcss-import` (everything should I think) and before autoprefixer. 

Perhaps add some extra features like import to the fixtures.